doc_template/examples_root/examples/biophysical_sim/biophysical_sim.py

```diff
@@ -0,0 +1,125 @@
+from allensdk.model.biophysical import runner,_schemas
+import json
+import os
+from argschema import ArgSchemaParser
+from allensdk.api.queries.biophysical_api import BiophysicalApi
+from allensdk.core.nwb_data_set import NwbDataSet
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import shutil
+
+sns.set(style='whitegrid',font_scale=1.5)
+plt.rcParams.update({'axes.grid':False})
+
+#%% Define utility functions
+
+def get_manifest_args(args):
+    return runner.load_description(args)
+    
+
+def get_sweep_data(nwb_file,sweep_number):
+    nwb = NwbDataSet(nwb_file)
+    sweep = nwb.get_sweep(sweep_number)
+    stim_diff = np.diff(sweep['stimulus']*1e12)
```

Comment:
Since this is in an example, could you explain what these scale parameters are and why they are needed? Maybe add them as kwargs to this function and write explanations in the docstring